tests/urdna2015/test060-in.nq

```diff
@@ -1,17 +1,38 @@
-<urn:ex:s> <urn:ex:000:empty> "" .
-<urn:ex:s> <urn:ex:001:simple> "simple" .
-<urn:ex:s> <urn:ex:002:quote> "\"" .
-<urn:ex:s> <urn:ex:003:backslash> "\\" .
-<urn:ex:s> <urn:ex:004:nl> "\n" .
-<urn:ex:s> <urn:ex:005:cr> "\r" .
-<urn:ex:s> <urn:ex:006:all> "\"\\\n\r" .
-<urn:ex:s> <urn:ex:007:uchar> "\u0022\u005c" .
-<urn:ex:s> <urn:ex:008:echar> "\t\b\n\r\f\"\'\\" .
-<urn:ex:s> <urn:ex:009> "\\u0039" .
-<urn:ex:s> <urn:ex:010> "\\n" .
-<urn:ex:s> <urn:ex:011> "\\\\" .
-<urn:ex:s> <urn:ex:012> "\"\"" .
-<urn:ex:s> <urn:ex:013> "\\\\\\" .
-<urn:ex:s> <urn:ex:014> "\"\"\"" .
-<urn:ex:s> <urn:ex:015> "\u221e" .
-<urn:ex:s> <urn:ex:016> "∞" .
+<_:s⁰> <_:p⁰> <_:o⁰> <_:g⁰> .
```

Comment:
These aren't valid IRIs, as `_` is not a valid IRI scheme. did you intend it to be `<_:s⁰> _:p⁰ _:o⁰ _:g⁰ .`? Although, note that a predicate can't be a blank node.

Reply:
Hmm.  I guess the js code I wrote should have not let that through.  A candidate for a negative n-quads test.

Reply:
What was this trying to test that the existing `<urn:ex:s:000:s⁰2>` line does not?  Should we just drop the line?